dotCMS/src/main/java/com/dotmarketing/db/LocalTransaction.java

```diff
@@ -6,6 +6,37 @@
 
 public class LocalTransaction {
 
+    static public <T> T wrapHibernateReturn(final ReturnableDelegate<T> delegate) throws Exception {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)